draft-ietf-trans-rfc6962-bis.md

```diff
@@ -2105,6 +2105,50 @@ details in this registry.
 Since log operators can choose to not use this registry (see {{log_id}}), it is
 not expected to be a global directory of all logs.
 
+## URN Sub-namespace for TRANS errors (urn:ietf:params:trans:error)
+
+IANA is requested to add a new entry in the
+"IETF URN Sub-namespace for Registered Protocol Parameter Identifiers"
+registry, following the template in {{!RFC3553}}:
+
+Registry name: trans:error
+
+Specification: RFCXXXX
+
+Repository: https://www.iana.org/assignments/trans
+
+Index value: No transformation needed.
+
+### TRANS Error Types
+
+IANA is requested to create a new registry for errors.
+Requirements for this registry are specification required.
+
+This registry should have the following three fields:
+
+|------------+--------+-----------|
+| Field Name | Type   | Reference |
+|------------+--------+-----------|
+| identifier | string | RFCXXXX   |
+| meaning    | string | RFCXXXX   |
+| reference  | string | RFCXXXX   |
+|------------+--------+-----------|
+
+The initial values are as follows, taken from the text above:
+
+|------------+----------------------------------+-----------|
+| Identifier | Meaning                          | Reference |
+|------------+----------------------------------+-----------|
+| malformed  | The request could not be parsed. | RFCXXXX   |
+|------------+----------------------------------+-----------|
+| badSubmission | `submission` is neither a valid certificate nor a valid precertificate        | RFCXXXX |
+| badType       | `type` is neither 1 nor 2     | RFCXXXX |
+| badChain      | The first element of `chain` is not the certifier of the `submission`, or the second element does not certify the first, etc. | RFCXXXX |
+| badCertificate | One or more certificates in the `chain` are not valid (e.g., not properly encoded) | RFCXXXX |
+| unknownAnchor  | The last element of `chain` (or, if `chain` is an empty array, the `submission`) both is not, and is not certified by, an accepted trust anchor | RFCXXXX |
+| shutdown       | The log is no longer accepting submissions | RFCXXXX |
+|---------------+----------------------------------|
```

Comment:
This covers the error codes from sections 5 and 5.1, but misses several additional error codes defined in other 5.x sub-sections.